src/System.Diagnostics.EventLog/dir.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
```

Comment:
Please change this to 4.0.0.0. Note- after you do this, you may have to build from root or your tests will start to not work.